From b581ad6a01960fab69352ba2e9f7a58c82a63c46 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 16 Aug 2021 18:59:43 -0400
Subject: [PATCH 52/78] disable search engine permissions by default

Disable the geolocation and notification permissions for the default
search engine by default.
---
 components/permissions/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/permissions/features.cc b/components/permissions/features.cc
index 390de2f97e2df..47555f270a898 100644
--- a/components/permissions/features.cc
+++ b/components/permissions/features.cc
@@ -76,7 +76,7 @@ const base::Feature kRevisedOriginHandling{"PermissionsRevisedOriginHandling",
 // When enabled, the Default Search Engine does not automatically receive the
 // "geolocation" and "notifications" permissions. DSE only applies to Android.
 const base::Feature kRevertDSEAutomaticPermissions{
-    "RevertDSEAutomaticPermissions", base::FEATURE_DISABLED_BY_DEFAULT};
+    "RevertDSEAutomaticPermissions", base::FEATURE_ENABLED_BY_DEFAULT};
 #endif  // defined(OS_ANDROID)
 
 }  // namespace features
-- 
2.34.1

